src/popup/task/task_view.js

```diff
@@ -122,7 +139,10 @@ const TaskView = View.extend({
       estimated_hours: this.estimate.value
     });
 
-    const account = this.accounts.get(this.user.value.account);
+    if (this.model.collection) {
+      this.model.collection.remove(this.model);
```

Comment:
Why is the model removed from the collection here?

Reply:
Hack to fix this issue:
1. Try save a task in a workspace you have readonly perms/network error for some reason.
2. Switch to a workspace you can write to.
3. The task's `parent` was not being updated.

We should probably only load accounts a user has write access to i guess...

Reply:
Good point, I will write it down for future consideration